README.md

```diff
@@ -120,10 +120,29 @@ async execute() {
 }
 ```
 
-#### `@Command.String({required?: boolean})`
+#### `@Command.String({required?: boolean, booleanIfNotBound?: boolean})`
```

Comment:
I'd suggest to name it `tolerateBoolean` (default `false` instead of `true`) to avoid double negations.

Reply:
Sounds good!